src/config/settings.ts

```diff
@@ -333,6 +354,11 @@ const settings: Settings = {
         ? (<string>process.env.BLOCKCHAINS_ETHEREUM_PROVIDER_URL).split(',')
         : [],
     },
+    sepolia: {
```

Comment:
this is not needed, please remove there is general method to fetch urls

Reply:
Removed